pulp_rpm/app/modelresource.py

```diff
@@ -490,7 +491,27 @@ class Meta:
         )
 
 
+class RpmRepositoryResource(RepositoryResource):
+    """
+    A resource for importing/exporting RPM repository entities.
+    """
+
+    def set_up_queryset(self):
+        """
+        Set up a queryset for RpmRepositories.
+
+        Returns:
+            A queryset containing one repository that will be exported.
+        """
+        return RpmRepository.objects.filter(pk=self.repo_version.repository)
+
+    class Meta:
+        model = RpmRepository
+        exclude = RepositoryResource.Meta.exclude + ("most_recent_version",)
```

Comment:
Same here, could you point me to what `most_recent_version` is referencing

Reply:
"most_recent_version" is a pulp_rpm Thing, that we don't recreate at load-time - it gets set, on the import-side, by the act of creating a new repo-version and filling it with the imported content. We don't export this value, because it's "owned", if you will, by the downstream repo.